mapswipe_workers/mapswipe_workers/project_types/base/project.py

```diff
@@ -217,7 +223,10 @@ def save_to_firebase(self, project, groups, groupsOfTasks):
                     json.dumps(tasks_list).replace(" ", "").replace("\n", "")
```

Comment:
I'm wondering if the `replace` calls you make here could remove spaces we actually want (like in a value). It might be preferable to use the `json.dumps()` argument `separator=(',', ':')` which ensures no extra spaces are added in the resulting string, unless you're positively sure we don't need any spaces anywhere in the result?

Reply:
the task attribute we have don't care about spaces, we use `taskId`, `geojson` and `center` (for Google TMS). For the tutorial additionally there are `screen` and `reference`. All these don't use/need spaces.